src/ui/public/field_format_editor/editors/url/url.html

```diff
@@ -8,6 +8,16 @@
   </select>
 </div>
 
+<div class="checkbox" ng-if="editor.formatParams.type == 'a'">
+  <label>
+    <input
+      type="checkbox"
+      ng-model="editor.formatParams.openLinkInCurrentTab">
```

Comment:
Can we move the `>` to the next line? Along the lines of [these examples](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md#multiple-attribute-values)